app/uk/gov/hmrc/exports/scheduler/jobs/emails/SendEmailsJob.scala

```diff
@@ -97,20 +97,29 @@ class SendEmailsJob @Inject() (
 
       case Success(MissingData) =>
         logger.warn(s"Could not send email because some data is missing")
-        failSingle(workItem)
+        failOrCancelSingle(workItem)
 
       case Success(BadEmailRequest(msg)) =>
         logger.warn(s"Email service returned Bad Request response: [$msg]")
-        failSingle(workItem)
+        failOrCancelSingle(workItem)
 
       case Success(InternalEmailServiceError(msg)) =>
         logger.warn(s"Email service returned Internal Service Error response: [$msg]")
         logger.warn(s"Will try again in ${interval.toMinutes} minutes")
-        failSingle(workItem)
+        failOrCancelSingle(workItem)
     }
 
-  private def failSingle(workItem: WorkItem[SendEmailDetails]): Future[Unit] =
-    sendEmailWorkItemRepository.markAs(workItem.id, Failed).flatMap { _ =>
+  private def failOrCancelSingle(workItem: WorkItem[SendEmailDetails]): Future[Unit] = {
```

Comment:
Can we pass to this method the `log.warn` messages from the calling method above so that it can output all the log messages? 

Reason for this change is that atm the cause of the processing failure will appear on a different log line than the error msg (and at a different log level) so disconnected and possible hidden from the reader (by a filter on the log level in kibana) .